account_cutoff_base/models/account_cutoff.py

```diff
@@ -0,0 +1,359 @@
+# -*- coding: utf-8 -*-
+# © 2013-2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields, api, _
+from openerp.exceptions import UserError
+
+
+class AccountCutoff(models.Model):
+    _name = 'account.cutoff'
+    _rec_name = 'cutoff_date'
+    _order = 'cutoff_date desc'
+    _inherit = ['mail.thread']
+    _description = 'Account Cut-off'
+
+    @api.multi
+    @api.depends('line_ids')
+    def _compute_total_cutoff(self):
+        for cutoff in self:
+            tamount = 0.0
+            for line in cutoff.line_ids:
+                tamount += line.cutoff_amount
+            cutoff.total_cutoff_amount = tamount
+
+    @api.model
+    def _default_move_label(self):
+        type = self._context.get('type')
+        label = ''
+        if type == 'accrued_expense':
+            label = _('Accrued Expense')
+        elif type == 'accrued_revenue':
+            label = _('Accrued Revenue')
+        elif type == 'prepaid_revenue':
+            label = _('Prepaid Revenue')
+        elif type == 'prepaid_expense':
+            label = _('Prepaid Expense')
+        return label
+
+    @api.model
+    def _inherit_default_cutoff_account_id(self):
+        '''Function designed to be inherited by other cutoff modules'''
```

Comment:
If docstring should really use double triple-quota, then why flake8/pylint says nothing about it ?